modules/ListView.js

```diff
@@ -59,10 +59,10 @@
       var childView = this.getChildView(model);
       if (childView) {
         childView.dispose();
-        this.unregisterChildView(childView);
+        this.unregisterTrackedView(childView, { shared: false });
```

Comment:
all these places you have shared:false are just bookkeeping right? it's not necessary?